components/x-gift-article/src/v2/FooterMessage.jsx

```diff
@@ -0,0 +1,96 @@
+import { h } from '@financial-times/x-engine'
+import { ShareType } from '../lib/constants'
+
+export const FooterMessage = ({
+	shareType,
+	isGiftUrlCreated,
+	redemptionLimit,
+	enterpriseLimit,
+	giftCredits,
+	isNonGiftUrlShortened,
+	enterpriseEnabled,
+	enterpriseRequestAccess,
+	includeHighlights
+}) => {
+	// if the share link has not been created
+	if (!isGiftUrlCreated && !isNonGiftUrlShortened) {
```

Comment:
**Suggestion**:
```
if (!(isGiftUrlCreated || isNonGiftUrlShortened))
```
is an alternative way of creating that if statement

Reply:
see [f5e1171](https://github.com/Financial-Times/x-dash/pull/713/commits/f5e117119252b4722b0682ca410e5d94e4186b1b)